frontend/src/views/LoginView.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <AppHeroImage />
   <v-container class="login-container" fluid v-bind="$attrs">
+    <AppAlertBanner v-for="message in systemMessages" type="info">{{ message.content }}</AppAlertBanner>
```

Comment:
This v-for requires a key. I don't think there's a natural key so you could add a guid after retrieval.

Reply:
it's updated